docs/Quick Reference.md

```diff
@@ -23,11 +23,11 @@ This table summarizes the filters and other options available inside a `tasks` b
 | `is recurring`<br>`is not recurring`                                                                                                                                                                                                                                                            | `sort by recurring`                         | `group by recurring`   |                        |
-| `recurrence (includes, does not include) <string>`<br>`recurrence (regex matches, regex does not match) /regex/i`                                                                                                                                                                               |                                             | `group by recurrence`  | `hide recurrence rule` |
-| **[[Filters#Priority\|Priority]]** and **[[Urgency\|urgency]]**                                                                                                                                                                                                                                          |                                             |                        |                        |
-| `priority is (above, below, not)? (low, none, medium, high)`                                                                                                                                                                                                                                    | `sort by priority`                          | `group by priority`    | `hide priority`        |
+| `recurrence (includes, does not include) <string>`<br>`recurrence (regex matches, regex does not match) /regex/i`
+| **[[Filters#Priority\|Priority]]** and **[[Urgency\|urgency]]**
+| `priority is (above, below, not)? (lowest, low, none, medium, high, highest)`                                                                                                                                                                                                                   | `sort by priority`                          | `group by priority`    | `hide priority`        |
 |                                                                                                                                                                                                                                                                                                 | `sort by urgency`                           | `group by urgency`     | `show urgency`         |
-| **[[Filters#Filters for File Properties\|File properties]]**                                                                                                                                                                                                                                    |                                             |                        |                        |
+| **[[Filters#Filters for File Properties\|File properties]]**
```

Comment:
It looks like the `|` for later columns has disappeared.

Reply:
should be fine now